### PR TITLE
Localize site name/description for metadata preview pane usage

### PR DESCRIPTION
**Pull request by wiz**
Allows you to localize the preview text on Slack, Twitter, etc. like this:

Screen Shot 2019-09-06 at 10 02 08

### CODE REVIEW

**Review by erciccione — APPROVED** [review]
utACK

**Comment by huey735** [comment]
@m52go  Could we change the English text to:
site_desc: "Bisq is an open-source desktop application that allows you to buy and sell bitcoins in exchange for national currencies or altcoins."

pt-PT translation
site_name: "Bisq - A exchange de Bitcoin descentralizada"
site_desc: "Bisq é uma aplicação de desktop open-source que lhe permite comprar e vender bitcoins em troca de moedas nacionais ou altcoins."

**Comment by wiz** [comment]
why not "alternative crypto-currencies" ? too long?

**Comment by m52go** [comment]
@wiz yeah for such site previews, every character counts. Would be too long, I think.

I would say it could be shortened even more to "Bisq is an open-source desktop application that allows you to trade bitcoin for national currencies or altcoins."

**Comment by wiz** [comment]
The max length for opengraph description is 155 characters. The string "Bisq is an open-source desktop application that allows you to buy and sell bitcoins in exchange for national currencies, or alternative crypto currencies." is 154 characters so it fits perfectly, and I don't see any issue?

**Comment by m52go** [comment]
No issues, but I would still say shorter is better, like how a 50-character tweet is more likely to be read entirely than a 240-character tweet.

I don't have a strong opinion here...it was just a suggestion. Maybe others can chime in.

**Comment by wiz** [comment]
Of course, you need to fit your translated text within the 155 character limit, so by all means shorten your translations if necessary. I think it's important to say "alternative crypto-currency" and not "altcoins" because newbies might not know what those are.

**Review by m52go — CHANGES_REQUESTED** [review]
test

EDIT: I suggested some changes but they didn't get added to my review. Will try to submit review again.

File: _data/languages.yml [hunk 1]
```diff
@@ -2,27 +2,41 @@
...
   enabled: No
   accept_languages: ["en", "en-us", "en-au", "en-nz", "en-za", "en-bz", "en-tt"]
+  site_name: "Bisq - The decentralized Bitcoin exchange"
+  site_desc: "Bisq is an open-source desktop application that allows you to buy and sell bitcoins in exchange for national currencies, or alternative cryptocurrencies."
 - name: "Deutsch"
   tag: "de"
   enabled: Yes
   accept_languages: ["de"]
+  site_name: "Bisq - The decentralized Bitcoin exchange"
+  site_desc: "Bisq is an open-source desktop application that allows you to buy and sell bitcoins in exchange for national currencies, or alternative cryptocurrencies."
```
Comment (m52go) [inline _data/languages.yml:12]:
```suggestion
  site_desc: "Bisq is an open-source desktop application that allows you to trade bitcoin for national currencies or alternative cryptocurrencies."
```

**Comment by wiz** [comment]
you can't suggest changes if I already committed and pushed them :wink:

File: _data/languages.yml [hunk 2]
```diff
@@ -2,27 +2,41 @@
...
 - name: "Español"
   tag: "es"
   enabled: Yes
   accept_languages: ["es", "es-mx", "es-gt", "es-cr", "es-pa", "es-do", "es-ve", "es-co", "es-pe", "es-ar", "es-ec", "es-cl", "es-uy", "es-py", "es-bo", "es-sv", "es-hn", "es-ni", "es-pr"]
+  site_name: "Bisq - The decentralized Bitcoin exchange"
+  site_desc: "Bisq is an open-source desktop application that allows you to buy and sell bitcoins in exchange for national currencies, or alternative cryptocurrencies."
```
Comment (m52go) [inline _data/languages.yml:18]:
```suggestion
  site_desc: "Bisq is an open-source desktop application that allows you to trade bitcoin for national currencies or alternative cryptocurrencies."
```

File: _data/languages.yml [hunk 3]
```diff
@@ -2,27 +2,41 @@
   tag: "en"
   enabled: No
   accept_languages: ["en", "en-us", "en-au", "en-nz", "en-za", "en-bz", "en-tt"]
+  site_name: "Bisq - The decentralized Bitcoin exchange"
+  site_desc: "Bisq is an open-source desktop application that allows you to buy and sell bitcoins in exchange for national currencies, or alternative cryptocurrencies."
```
Comment (m52go) [inline _data/languages.yml:6]:
```suggestion
  site_desc: "Bisq is an open-source desktop application that allows you to trade bitcoin for national currencies or alternative cryptocurrencies."
```

**Review by m52go — COMMENTED** [review]
I prefer this wording.

**Comment by wiz** [comment]
IMO your requested changes are out of scope for this PR. This PR is only to implement the ability to localize the description into various languages, not to change any strings. If you want to change strings, you should do that in your own PR, not in mine :)

**Comment by m52go** [comment]
You're on thin ice here buddy. But I acknowledge this contribution 😄

**Comment by m52go** [comment]
ACK

**Pull request merged**